static/js/portico/signup.js

```diff
@@ -22,6 +22,11 @@ $(() => {
         });
     }
 
+    common.toggle_password("#ldap-password", ".password_visibility_toggle1");
+    common.toggle_password("#id_password", ".password_visibility_toggle2");
+    common.toggle_password("#id_new_password1", ".password_visibility_toggle_one");
+    common.toggle_password("#id_new_password2", ".password_visibility_toggle_two");
```

Comment:
`password_visibility_toggle1` vs. `password_visibility_toggle_one` is confusing. The names should clearly indicate the difference. Or we could use a sibling (`#ldap-password ~ .password_visibility_toggle`) or adjacent sibling (`#ldap-password + .password_visibility_toggle`) selector and avoid coming up with separate names.